XamlControlsGallery/ControlPages/MenuBarPage.xaml.cs

```diff
@@ -26,5 +26,19 @@ public MenuBarPage()
         {
             this.InitializeComponent();
         }
+
+        private void OnElementClicked(object sender, Windows.UI.Xaml.RoutedEventArgs e)
```

Comment:
All three of these events have the same logic. Why can't they be combined into a single OnElementClicked method?

Reply:
yes, I'm not sure how to get the parent element, I've tired doing 
```
var example = ((sender as MenuFlyoutItem).Parent as ControlExample);
            example.onClickedText = "You clicked: " + (sender as MenuFlyoutItem).Text;
```

Reply:
just for learning purposes, would you know how to successfully get the parent object of the ControlExample in this case? Thanks

Reply:
Walking up the parent tree is generally not advised, but there is some sample code in the following issue that shows how to do it. [microsoft-ui-xaml #1331](https://github.com/microsoft/microsoft-ui-xaml/issues/1331)

Doing so involves using `currParent = VisualTreeHelper.GetParent(child);` and recursing up the tree.

Reply:
🦙 [Or using the Toolkit](https://docs.microsoft.com/en-us/windows/communitytoolkit/extensions/visualtree) 😋